tests/strictEqual.test.js

```diff
@@ -0,0 +1,82 @@
+const describe = require('mocha').describe
+const it  = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictEqual')
+
+describe('strictEqual', () => { 
+    describe('strictEqual', () => { 
+        it('returns true if values provided are same and are of the same type',() => {
+            // create test data
+
+            const valueOne = '5' 
+            const valueTwo = '5'
+
+
+
+            // call the function with the test data
+
+            const Return = strictEqual(valueOne,valueTwo)
+
+            //make assertions about what should be true
+
+            expect(Return).to.equal(true) 
+        })
+        it('returns false if values provided are different and are of  the same type',() => {
```

Comment:
Try to clean up your spacing as you're writing code. The more code that fits on the screen (with suitable breathing room) the better. Something like this condenses things while still having breaks to separate logical pieces from each other such as calling the test function and creating an assertion.

```JavaScript
        it('returns true if values provided are same and are of the same type',() => {
            const valueOne = '5' 
            const valueTwo = '5'

            const Return = strictEqual(valueOne,valueTwo)

            expect(Return).to.equal(true) 
        })

        it('returns false if values provided are different and are of  the same type',() => {
```